crates/krun/src/bin/krun.rs

```diff
@@ -115,6 +117,24 @@ fn main() -> Result<()> {
         }
     }
 
+    if let Ok(run_path) = env::var("XDG_RUNTIME_DIR") {
+        let pulse_path = Path::new(&run_path).join("pulse/native");
+        if pulse_path.exists() {
+            let c_pulse_path = CString::new(
+                pulse_path
+                    .to_str()
+                    .expect("pulse_path should not contain invalid UTF-8"),
+            )
+            .expect("pulse_path should not contain NUL character");
```

Comment:
https://doc.rust-lang.org/std/env/fn.var.html#errors

> This function may return an error if the environment variable’s name contains the equal sign character (`=`) or the NUL character.

> may

I think it's safer for us to not rely on that? (We already return an error in other parts of the code.)

```suggestion
            . context("Failed to process `pulse/native` path as it contains NUL character")?;
```

I mean, of course this shouldn't happen in reality anyway. And I'm not sure if there's any meaningful difference in practice here.